docs/make.jl

```diff
@@ -65,10 +35,13 @@ makedocs(;
         "create_kernel.md",
         "API" => "api.md",
         "Design" => "design.md",
-        "Examples" =>
-            map(filter!(filename -> endswith(filename, ".md"), readdir(EXAMPLES_OUT))) do x
-                return joinpath("examples", x)
-            end,
+        "Examples" => map(
+            basename.(
+                filter!(isdir, readdir(joinpath(@__DIR__, "src", "examples"); join=true)),
+            ),
+        ) do x
+            joinpath("examples", x, "index.md")
+        end,
```

Comment:
@theogf given that JuliaGPsDocs is an opinionated package for making it easier to get consistent docs setup across our packages, how about we let it also provide this as a function so we can just do
```suggestion
        "Examples" => JuliaGPsDocs.find_notebook_examples(),
```
? :)

Reply:
https://github.com/JuliaGaussianProcesses/JuliaGPsDocs.jl/pull/15

Reply:
@theogf I've updated the suggestion above so that it should work now.